environment-dev.yml

```diff
@@ -5,7 +5,8 @@ dependencies:
   - pip
 
   # required dependencies
-  - pandas>=2.2,<2.3
+  # excluding 2.2.0 because of https://github.com/modin-project/modin/issues/6968
+  - pandas>=2.2,<2.3,!=2.2.0
```

Comment:
if a user have `pandas 2.2.0` installed and would try to execute `df.groupby("a").first(skipna=False)` on Modin, he will get an error thrown by pandas as this argument doesn't exist there. Not sure, if that's the reason to exclude the pandas version, what others think?

Reply:
https://github.com/pandas-dev/pandas/issues/57631